examples/custom-service.mdx

```diff
@@ -20,49 +38,117 @@ edition = "2021"
 [lib]
 
 [dependencies]
-shuttle-service = { version = "0.9.0" }
-shuttle-shared-db = { version = "0.9.0", features = ["postgres"] }
-sqlx = { version = "0.6", features = ["runtime-tokio-native-tls", "postgres"] }
+anyhow = "1.0.62"
+axum = "0.6.4"
+hyper = "0.14.24"
+poise = "0.5.2"
+serde = "1.0"
+shuttle-secrets = "0.9.0"
+shuttle-service = { version = "0.9.0", features = ["bot-serenity"] }
+tokio = "1.22.0"
 ```
 
-This section will explain how to create a custom shuttle service. To do this we need 
-to return a struct from our `shuttle_service::main` function that implements `shuttle_service::Service`.
-
-In this simple example we will implement `Service` for a `PgPool`, that will simply return 'Hello world'
-from the database on startup. In Rust we can't implement foreign traits for foreign types, so we have 
-to create a wrapper around our `PgPool`.
+### Getting Started
+To get started, we need to return a wrapper struct from our `shuttle_service::main` function that 
+implements `shuttle_service::Service`, which we can declare like so:
 
 ```rust
-struct PoolService {
-    pool: sqlx::PgPool,
+struct CustomService {
+    discord_token: String,
```

Comment:
Should we rather wrap the poise client/framework here, as well as the axum Router?

Reply:
This could also be just empty. Having a discord token is better than the pool IMO as it makes it clear it is purely application specific.